apps/platform/src/components/ui/combobox.tsx

```diff
@@ -37,19 +38,40 @@ import {
   DialogTrigger
 } from './dialog'
 import { Button } from './button'
+import { WorkspaceSchema } from '@keyshade/schema/schemas'
+
+interface WorkspaceResponse {
+  items: Workspace[];
+  "metadata": {
+    page: number;
+    perPage: number,
+    pageCount: number,
+    totalCount: number,
+    links: {
+        self: string,
+        first: string,
+        previous: string | null,
+        next: string | null,
+        last: string
+    }
+}
+}
 
 async function getAllWorkspace(): Promise<Workspace[] | undefined> {
   try {
-    const workspaceData: Workspace[] =
-      await apiClient.get<Workspace[]>('/workspace')
-
-    const { success, data } = zWorkspace.array().safeParse(workspaceData)
+    const workspaceData: WorkspaceResponse =
+      await apiClient.get<WorkspaceResponse>('/workspace')
```

Comment:
You shouldnt be using direct calls. We have the controllers defined in `api-client`

Reply:
> You shouldnt be using direct calls. We have the controllers defined in `api-client`

This was done intentionally, @kriptonian1 was with me. There were some issues from the `api` side on getting the `currentWorkspace`. That's why this was done, just to make sure that the flow atleast works.

Reply:
Oh, but do keep a TODO: in here to change it later on.